roles/openshift_logging/README.md

```diff
@@ -12,7 +10,156 @@ generation for Elasticsearch (it uses JKS) as well as openssl to sign certificat
 As part of the installation, it is recommended that you add the Fluentd node selector label
 to the list of persisted [node labels](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-node-host-labels).
 
-### Required vars:
+### Change to variable structure and deployment structure:
+
+Previously, the EFK stack was deployed as either an all-in-one or a split (apps and ops) deployment based on the boolean variable
+`openshift_logging_use_ops`. This has been phased out in favor of being able to configure multiple (currently up to two is supported)
+ES clusters.
+
+To be able to accomplish this, the old variable naming structure has been revamped to follow a dictionary structure as such:
+
+```yaml
+openshift_logging:
+  cluster_names: ['es']
+  fluentd:
+    nodeselector: {'logging-infra-fluentd': 'true'}
+    cpu_limit: null
+    cpu_request: 100m
+    memory_limit: 512Mi
+    journal_source: ""
+    journal_read_from_head: ""
+    hosts: ['--all']
+    buffer_queue_limit: 32
+    buffer_size_limit: 8m
+    app:
+      host: logging-es
+      port: 9200
```

Comment:
I think port is same for both app and infra, maybe not needed here?

Reply:
Yeah, this was a copy-paste from the defaults. I'll likely update this to make more sense since the default for `infra.port` should be `{{ app.port }}`